### PR TITLE
[13.0][ADD] product_abc_classification_sale: New module to classify products from sale orders

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31774

This module depends of: 

- [x] https://github.com/OCA/product-attribute/pull/1013

### CODE REVIEW

File: product_abc_classification_sale/views/product_views.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+    <record id="product_template_only_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
```
Comment (MiquelRForgeFlow) [inline product_abc_classification_sale/views/product_views.xml:4]: why not add `<field name="name">`?

**Review by MiquelRForgeFlow — COMMENTED** [review]
@CarlosRoca13 Have you rebased properly? checking travis error seems like last update in product_abc_classification is not taken.

File: product_abc_classification_sale/views/product_views.xml (hunk 1 above)
Comment (CarlosRoca13) [reply product_abc_classification_sale/views/product_views.xml:4]: It is not necessary, it will be computed by the system

**Review by chienandalu — APPROVED** [review]
Functional and code review :+1: Thanks!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**